src/platform/plugins/private/kibana_overview/public/components/overview/overview.tsx

```diff
@@ -76,6 +79,12 @@ export const Overview: FC<Props> = ({ newsFetchResult, solutions, features }) =>
   } = services;
   const addBasePath = http.basePath.prepend;
   const currentTheme = useObservable(theme.theme$, { darkMode: false, name: 'amsterdam' });
+  const { euiTheme } = useEuiTheme();
+  const minBreakpointM = useEuiMinBreakpoint('m');
+
+  const className = classNames({
```

Comment:
nit? 

```suggestion
  const newsContainerClassName = classNames({
```

or just make it inline?